java.hints/src/org/netbeans/modules/java/hints/jdk/ConvertToVarHint.java

```diff
@@ -209,17 +243,15 @@ private static boolean isValidVarType(HintContext ctx) {
                 default:
                     break;
             }
+            // variable initializer type should be same as variable type.
```

Comment:
Please try to only include the changes required for the fix in the transaction. For formatting or code style changes in existing code you can create a new transaction. It makes it hard to compare code to see the actual changes.